processing/src/main/java/org/apache/druid/query/aggregation/last/StringLastAggregatorFactory.java

```diff
@@ -302,6 +303,7 @@ public String toString()
            "fieldName='" + fieldName + '\'' +
            ", name='" + name + '\'' +
            ", maxStringBytes=" + maxStringBytes +
```

Comment:
```suggestion
           ", maxStringBytes=" + maxStringBytes + '\'' +
```

Reply:
I think we don't necessarily need to quote this; as its an integer
I see other places where the same value is shown in `toString` and its [not quoted there](https://github.com/apache/druid/blob/647686aee2a0cf29cc50a13c53d97b61ba479e0c/processing/src/main/java/org/apache/druid/query/aggregation/any/StringAnyAggregatorFactory.java#L225) 
but in case we do; we should quote it from both sides

Reply:
Blahh that's an integer, I'll merge this